### PR TITLE
Add first pass at guard support

### PR DESCRIPTION
**Pull request by danhodos**
This was cribbed from the [guard-hologram][1] gem and the guard wiki's [Create a Guard page][2].

[1]: https://github.com/kmayer/guard-hologram
[2]: https://github.com/guard/guard/wiki/Create-a-guard

### CODE REVIEW

**Comment by wtfluckey** [comment]
:

File: lib/guard/txi_rails_hologram.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  class TxiRailsHologram < Plugin
+    include TxiRailsHologramVersion
+    # Called when just `enter` is pressed
+    # This method should be principally used for long action like running all specs/tests/...
+    #
+    # @raise [:task_has_failed] when run_all has failed
+    # @return [Object] the task result
+    #
+    def run_all
+      system *"bundle exec rake txi_rails_hologram:build".split or throw :task_has_failed
```
Comment (theotherzach) [inline lib/guard/txi_rails_hologram.rb]: What would you think of `system * %w{bundle exec rake txi_rails_hologram:build}`?

Comment (theotherzach) [inline lib/guard/txi_rails_hologram.rb]: This is obviously a nitpick reflecting personal taste.. Feel free to ignore.

**Comment by theotherzach** [comment]
:+1:

File: lib/guard/txi_rails_hologram.rb (hunk 1 above)
Comment (danhodos) [inline lib/guard/txi_rails_hologram.rb]: Hopefully I won't regret editing this from my phone on the train, but I've simplified this from he version cribbed from guard-hologram.

**Comment by danhodos** [comment]
Merging, as @ebinion and I are pairing and would like this. :D

**Pull request merged**